src/hipscat/catalog/association_catalog/association_catalog.py

```diff
@@ -32,10 +35,11 @@ def __init__(
         join_pixels: JoinPixelInputTypes,
         catalog_path=None,
         storage_options: Union[Dict[Any, Any], None] = None,
+        moc: MOC | None = None,
```

Comment:
Can you move the `moc` argument up in the list? We have a semi-convention of putting the `storage_options` as the last argument. Since neither is positional, the order doesn't *really* matter, but my eyes tend to stop reading after `storage_options`.

Here and elsewhere.